CodeGen/src/IrLoweringA64.cpp

```diff
@@ -703,6 +704,20 @@ void IrLoweringA64::lowerInst(IrInst& inst, uint32_t index, const IrBlock& next)
         build.fcsel(inst.regA64, temp1, inst.regA64, getConditionFP(IrCondition::Less));
         break;
     }
+    case IrCmd::SELECT_NUM:
+    {
+        LUAU_ASSERT(FFlag::LuauCodeGenLerp);
+        inst.regA64 = regs.allocReuse(KindA64::d, index, {inst.a, inst.b});
```

Comment:
Seems like 'c' and 'd' can also be the last use of this instruction and have the double type?
We just rarely see so many numeric inputs to an instruction.

Reply:
Yeah, similarly X64 can reuse a/c/d

Reply:
Updated both